polish docs (#23)
apache/skywalking-satellite: docs/en/guides/README.md

# Guides
If you want to debug or develop SkyWalking Satellite, The following documentations would guide you.

> - [How to contribute the plugins?](./contribuation/plugin.md)
> - [How to add unit test in the plugins?](./test/test.md)
> - [How to compile SkyWalking Satellite?](./compile/compile.md)

Reviewer: ```suggestion
- [How to contribute a plugin?](./contribuation/plugin.md)
- [How to add unit test for a plugin?](./test/test.md)
- [How to compile SkyWalking Satellite?](./compile/compile.md)
```
